src/routes/api/auth.js

```diff
@@ -20,6 +21,7 @@ router.post('api.auth.login', '/', async (ctx) => {
   if (!user) ctx.throw(404, `No user found with ${email}`);
   const authenticated = await user.checkPassword(password);
   if (!authenticated) ctx.throw(401, 'Invalid password');
+  await sendExampleEmail(ctx, ctx.request.body, user);
```

Comment:
Es mejor que dejemos esto desactivado en `main`, pues seguiremos utilizando la API creada, y que se envíe un correo cada vez que se hace login puede ser muy costoso, y algo limitante.

Para eso podrías utilizar una variable constante que definas al principio de este archivo, y que maneje si está activo o no el envío del correo. Inicialmente estaría seteada en `false`:
```javascript
const AUTH_MAILING_ACTIVE = false;
```

Y aquí  sólo envías el correo en caso que esté activa:
```suggestion
  if (AUTH_MAILING_ACTIVE) await sendExampleEmail(ctx, ctx.request.body, user);
```
De esta manera, si alguien con el código quisiera probar el envío de correos, sólo cambia el valor de la variable de `false` a `true`. Sin embargo, estará desactivado por defecto para evitar los problemas mencionados inicialmente.

Reply:
Excelente profesor! realizo los cambios entonces. 

Gracias